src/views/RecIM/components/List/Listing/index.js

```diff
@@ -269,9 +269,30 @@ const MatchListing = ({ match, activityID }) => {
         className={styles.listing}
       >
         <Grid container>
-          <Grid item>
-            {match.Team[0]?.Name} vs. {match.Team[1]?.Name}
-          </Grid>
+          {match.Team.length > 2 ? (
+            match.Team.map((team) => (
+              <>
+                <Grid item xs={10}>
+                  {team.Name}
+                </Grid>
+                <Grid item xs={2} textAlign="center">
+                  vs.
+                </Grid>
+              </>
+            ))
+          ) : (
```

Comment:
What happens to the last team in that list?
wouldn't it look like:

Team A vs Team B vs Team C vs
?

Also wouldn't this be an incredibly long list of names? I argue we just throw a:
<Typography style={styles.subtitle} >
Multiple Teams
</Typography>